index.html

```diff
@@ -23,14 +23,20 @@
         <img src="link.svg" alt="External link icon">
       </section>
     </a>
-    <a href="values">
+    <a href="https://medium.com/firefox-ux">
       <section style="background-color: var(--blue-70)">
+        <p>Articles</p>
+        <img src="link.svg" alt="External link icon">
+      </section>
+    </a>
+    <a href="values">
+      <section style="background-color: var(--blue-80)">
         <p>Design Values</p>
         <img src="link.svg" alt="External link icon">
       </section>
     </a>
     <a href="https://careers.mozilla.org/listings/?team=User+Experience">
-      <section style="background-color: var(--blue-80);">
+      <section style="background-color: var(--blue-90);">
```

Comment:
This also seems like something we could do a little better…  Inline styles for the fail.  😉